src/main/resources/templates/loginRequestedScope.html

```diff
@@ -13,7 +13,9 @@
     <div class="jumbotron jumbotron-fluid">
       <div class="container">
         <h1 class="display-6">Proof of concept application</h1>
-        <p class="lead">Enter the requested scopes you wish to obtain.</p>
+        <p class="lead">Enter the requested scopes you wish to obtain. To file with Companies House you must also obtain the user_profile : read scope.
```

Comment:
`user_profile : read` is a permission, not a scope. Should we refer to a valid scope instead?

Would it be a good idea prepopulating the scope input field with that value?

What happens if the user does not enter that scope? Should we handle that scenario?

Reply:
Changed the text to be more accurate and prepopulated the text area with the relevant scope. If they do not enter that scope then they haven't read the text area correctly.